var/spack/repos/builtin/packages/portcullis/package.py

```diff
@@ -45,6 +45,11 @@ def patch(self):
             'scripts/Makefile.am', string=True
         )
 
+        # remove -m64 on aarch64
+        if 'aarch64' in self.spec.architecture.target.lower():
```

Comment:
as in the other PR: maybe a `if not 'amd64' in ...` is even better?

Reply:
`if spec.satisfies(target='aarch64'):` would be preferable.

Reply:
The -m64 option is supported on least x86, sparc and powerPC:
https://gcc.gnu.org/onlinedocs/gcc-9.1.0/gcc/Option-Summary.html#Option-Summary
So I don't think if not x86_64 is better.

Reply:
> `if self.spec.satisfies(target='aarch64'):` would be preferable.

I rewrite so.